collector/mssql.go

```diff
@@ -662,6 +666,12 @@ func NewMSSQLCollector() (Collector, error) {
 			[]string{"instance"},
 			nil,
 		),
+		AccessMethodsWorktablesFromCacheRatio_Base: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "accessmethods_worktables_from_cache_ratio_lookups"),
```

Comment:
Skip the `_ratio` here, since it is not lookups on the ratio

Reply:
Also, the metric `accessmethods_worktables_from_cache_ratio` should probably be `accessmethods_worktables_from_cache_hits`, right?

Reply:
_(The above seems to repeat for all the new changes)_

Reply:
The bases have no meaning on itself if you go by the Microsoft documentation.    
Better said MS is not sharing  what they are in their [documentation](https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-buffer-manager-object?view=sql-server-ver15).    
Because of this I would prefer the naming style of @sqlkabouter .

We can't be sure here as we have to go with the documentation of Microsoft.

Reply:
I'm sorry to be a bit contrarian here, but while I appreciate that the documentation is a bit vague and the current naming wrong, I'm not sure I'm keen on merging something which introduces _more_ unclarity. The name here is pretty bad in that regard, in my opinion.
Looking at the docs you linked:
>The ratio is the total number of cache hits divided by the total number of cache lookups over the last few thousand page accesses.

By that definition, it seems pretty clear to me that we get the total lookups in the base, and total hits in the "non-base" counter? Are there other reasonable interpretations I'm missing?

Reply:
It seems to be the case after testing with MS monitoring tools.    
Our DB team will hopefully confirm with this Snapshot that the results using Prometheus+WMI_Exporter and MS tools report the same stats.